app/models/miq_action.rb

```diff
@@ -742,7 +742,7 @@ def action_container_image_annotate_deny_execution(action, rec, inputs)
       return
     end
 
-    unless rec.digest.present?
+    unless rec.kind_of?(ManageIQ::Providers::Openshift::ContainerManager::ContainerImage)
```

Comment:
Please check presence of the annotate method, since that is what we are going to do here